browser/src/core/Debug.js

```diff
@@ -863,25 +861,23 @@ L.DebugManager = L.Class.extend({
 
 	setOverlayMessage: function(id, message) {
 		if (this.overlayOn) {
-			if (!this._overlayData[id]) {
-				var topLeftNames = ['tileData', 'eventDelayTime'];
-				var position = topLeftNames.includes(id) ? 'topleft' : 'bottomleft';
-				this._overlayData[id] = L.control.attribution({prefix: '', position: position});
-				this._overlayData[id].addTo(this._map);
-			}
-			this._overlayData[id].setPrefix(message);
+			this._overlayData[id] = message;
 		}
 	},
 
 	clearOverlayMessage: function(id) {
-		if (this.overlayOn) {
-			if (this._overlayData[id]) {
-				this._overlayData[id].remove();
-				delete this._overlayData[id];
-			}
+		if (this.overlayOn && this._overlayData[id]) {
+			delete this._overlayData[id];
 		}
```

Comment:
Same here. Nit-picking of course.

Reply:
I think make prettier does this for me (?)